label_studio/data_manager/api.py

```diff
@@ -255,11 +255,15 @@ def get(self, request):
             # keep ids ordering
             page = [tasks_by_ids[_id] for _id in ids]
 
+            # TODO MM TODO this needs a discussion, because I'd expect
+            # people to retrieve manually instead on DM load, plus it
+            # will slow down initial DM load
+
             # retrieve ML predictions if tasks don't have them
-            if not review and project.evaluate_predictions_automatically:
-                tasks_for_predictions = Task.objects.filter(id__in=ids, predictions__isnull=True)
-                evaluate_predictions(tasks_for_predictions)
-                [tasks_by_ids[_id].refresh_from_db() for _id in ids]
+            # if not review and project.retrieve_predictions_automatically:
+            #     tasks_for_predictions = Task.objects.filter(id__in=ids, predictions__isnull=True)
+            #     retrieve_predictions(tasks_for_predictions)
+            #     [tasks_by_ids[_id].refresh_from_db() for _id in ids]
```

Comment:
What's the outcome of the discussion here? looks like another place where we used to consider `project.retrieve_predictions_automatically` but don't anymore. In the final version, we shouldn't have commented out code here.